app/scripts/controllers/users/list.js

```diff
@@ -35,7 +36,7 @@ angular.module('flightNodeDemo')
 							return {
 								fullName: user.givenName + ' ' + user.familyName,
 								email: user.email,
-								phone: (user.mobilePhoneNumber || user.phoneNumber),
+								phone: (user.phoneNumber || user.mobilePhoneNumber),
```

Comment:
Oops, I need to change that to `primaryPhoneNumber`, and then no need for secondary since primary is required. Although the database model still has `phoneNumber` and `mobilePhoneNumber`, I'm trying to move away from the idea of assuming that the main phone number is not the mobile number, since so many of us no longer have a "home" number (landline).